src/SpotifyApi.NetCore/IFollowApi.cs

```diff
@@ -80,10 +80,10 @@ public interface IFollowApi
             );
 
         /// <summary>
-        /// Check to see if one or more Spotify users are following a specified playlist.
+        /// Check if Users Follow a Playlist
```

Comment:
Please keep the same text as the Spotify docs, i.e. "Check to see if one or more Spotify users are following a specified playlist."